content/v2.0/reference/flux/stdlib/contrib/rows/map.md

```diff
@@ -0,0 +1,223 @@
+---
+title: rows.map() function
+description: >
+  The `rows.map()` function is an alternate implementation of [`map()`](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+  that is faster, but more limited than `map()`.
+menu:
+  v2_0_ref:
+    name: rows.map
+    parent: Rows
+weight: 202
+v2.0/tags: [functions, package, map]
+related:
+  - /v2.0/reference/flux/stdlib/built-in/transformations/map/
+---
+
+The `rows.map()` function is an alternate implementation of [`map()`](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+that is faster, but more limited than `map()`.
+`rows.map()` cannot modify table groups key and, therefore, does not need to regroup tables.
```

Comment:
table group keys? Unless this is an "attorneys general" situation I'm not understanding haha.

Reply:
I removed "table" and added a link to the "group key" definition in the glossary.